pkg/operator/ceph/config/conditions.go

```diff
@@ -150,7 +214,7 @@ func CheckConditionReady(context *clusterd.Context, namespace, name string) stri
 	if conditionMap[cephv1.ConditionReady] == v1.ConditionTrue {
 		return "Cluster is checking if updates are needed"
 	}
-	return "Cluster is creating"
+	return "Cluster is checking mon's health"
```

Comment:
- The operator checks the mon's health every 45 seconds. So if we're updating the conditions every time, this would be noisy if someone is watching it. Or is this only done with the main operator code path that would be less frequent? 
- A specific message about checking mon health seems better for the mon code. This is a more generic code path it seems.

Reply:
`Cluster is checking mon's health` message will only show initially i.e when the cluster is creating. Whenever the operator is restarting or another reconcilation starts it will be `Cluster is checking if updates are needed`. I hope this is okay? @travisn

Reply:
If this message is only during the full reconciliation that sounds good.

Reply:
For the message I'd suggest: "Cluster is checking mon health"